.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+end_of_line = 1f
+insert_final_newline = true
+indent_style = space
+
+[*.php]
+indent_size = 4
```

Comment:
It's only for php files.

Then again I _should_ add indent_size for css and js